src/lib/viewers/controls/settings/SettingsFlyout.tsx

```diff
@@ -5,14 +5,20 @@ import './SettingsFlyout.scss';
 
 export type Props = React.PropsWithChildren<{
     className?: string;
+    disableTransitions?: boolean;
     isOpen: boolean;
 }>;
 
-export default function SettingsFlyout({ children, className, isOpen }: Props): JSX.Element {
+export default function SettingsFlyout({
+    children,
+    className,
+    disableTransitions = false,
+    isOpen,
+}: Props): JSX.Element {
     const [isTransitioning, setIsTransitioning] = React.useState(false);
     const flyoutElRef = React.useRef<HTMLDivElement>(null);
-    const { activeRect } = React.useContext(SettingsContext);
-    const { height, width } = activeRect || { height: 'auto', width: 'auto' };
+    const { activeRect = { height: 'auto', width: 'auto' } } = React.useContext(SettingsContext);
+    const { height, width } = disableTransitions ? { height: 'auto', width: 'auto' } : activeRect;
```

Comment:
Do we need this prop? The menu should be set to the height of its contents by default. I think we should try to minimize the number of options we support in these menus.